Makefile

```diff
@@ -74,3 +74,9 @@ NETWORKING_CALICO_CHECKOUT?=master
 networking-calico:
 	git clone $(NETWORKING_CALICO_REPO)
 	cd networking-calico && git checkout $(NETWORKING_CALICO_CHECKOUT)
+
+.PHONY: master calico-%
+master:
```

Comment:
How about `publish-master`, just to emphasise that this is going to push something to a public repo?

Reply:
In parallel with your review, I wondered about changing this to `make release VERSION=vX.Y.Z`, for compatibility with our component Makefiles?  I think that might address both of your comments here.  WDYT?

Reply:
It's `release-publish` in the other makefiles.  `make release` only builds the release

Reply:
Cool, I'll do release-publish...